lib/matplotlib/tests/test_axes.py

```diff
@@ -1968,6 +1968,27 @@ def get_next_color():
                     c=c_case, edgecolors="black", kwargs={}, xsize=xsize,
                     get_next_color_func=get_next_color)
 
+    @check_figures_equal(extensions=["pdf"])
```

Comment:
Wouldn't it be simpler to test `path_collection.get_linewidths()` on the `PathCollection` returned from `ax.scatter()` instead of an image comparison?

Reply:
Yes, that's a possibility. We already have plot/scatter cross-validation from `test_markers.py`.